### DISCUSSION TITLE
Seems to work on Ubuntu Linux (OBS 29.0.0)

### DISCUSSION THREAD

**Discussion by xurei** [MisutaaAsriel/obs-toolbar#7] [General]
I installed the plugin by extracting the content of the .deb and putting it in the `plugins` folder of my OBS configuration anhd it work like a charm :+1: 
BTW I think you should provide the linux package in a .zip aside of the .deb, so people on different distros can easily install it.

Thanks for your work !

**Comment by MisutaaAsriel** [comment]
I am unable to do so at this time due to how the Linux builds are compiled; GitHub Actions are used and I am not confident enough to modify the build system to change this on my own.
